charts/index.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v1
+entries:
+  azurefile-csi-driver:
+  - apiVersion: v1
+    appVersion: v0.7.0
+    created: "2020-05-25T02:02:09.483818313+05:30"
+    description: Azure File Container Storage Interface (CSI) Storage Plugin
+    digest: 1ab5f5bfaffd3eec060c7ab68d933d72becf721e1c9868b9292b716cc856a39a
+    home: https://github.com/kubernetes-sigs/azurefile-csi-driver
+    name: azurefile-csi-driver
+    sources:
+      - https://github.com/kubernetes-sigs/azurefile-csi-driver
+    urls:
+    - https://raw.githubusercontent.com/kubernetes-sigs/azurefile-csi-driver/master/charts/azurefile-csi-driver-v0.7.0.tgz
+    version: v0.7.0
+  - apiVersion: v1
+    appVersion: v0.6.0
+    created: "2020-05-25T02:02:09.483531499+05:30"
+    description: Azure File Container Storage Interface (CSI) Storage Plugin
+    digest: ad581478836e360fc6b8367791867e7184268857b4f50b322ef5245010768bd4
+    home: https://github.com/kubernetes-sigs/azurefile-csi-driver
```

Comment:
Hi @andyzhangx, I have not included maintainer details for now. 
```
maintainers:
    - email: <email>
      name: <username/name>
```

Let me know if you want me to include your details. Thanks :)

Reply:
here is the maintainer details, thanks.
```
maintainers:
    - email: xiazhang@microsoft.com
      name: andyzhangx
```